app/cdap/components/TriggeredPipelines/store/TriggeredPipelineStore.ts

```diff
@@ -66,7 +74,9 @@ const TriggeredPipelineStore = createStore(
   {
     triggered: defaultInitialState,
   },
-  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+  window &&
```

Comment:
is this for unit tests?

Reply:
It's for the ts migration. There will be error message `Property '__REDUX_DEVTOOLS_EXTENSION__' does not exist on type 'Window & typeof globalThis'.` if I don't cast it to any